.ecrc

```diff
@@ -0,0 +1,16 @@
+{
+  "Verbose": false,
+  "Debug": false,
+  "IgnoreDefaults": false,
+  "SpacesAftertabs": false,
+  "NoColor": false,
+  "Exclude": ["\/fixtures\/"],
+  "AllowedContentTypes": [],
+  "PassedFiles": [],
+  "Disable": {
+    "EndOfLine": false,
+    "Indentation": false,
+    "InsertFinalNewline": false,
+    "TrimTrailingWhitespace": false
+  }
+}
```

Comment:
What is ecrc?

Reply:
Configuration file for editorconfig: https://editorconfig-checker.github.io/

Reply:
We already have eslint and prettier. I don't think we want another formatting config. cc @lucleray

Reply:
I think prettier/eslint should be enough. We have `.editorconfig` as a "nice to have" but prettier/eslint are the "authority" for formatting/linting.

Reply:
Okay.